CHANGELOG.md

```
@@ -519,10 +528,16 @@
 [#2165]: https://github.com/THM-Health/PILOS/pull/2165
 [#2222]: https://github.com/THM-Health/PILOS/pull/2222
 [#2223]: https://github.com/THM-Health/PILOS/pull/2223
+<<<<<<< HEAD
 [#2265]: https://github.com/THM-Health/PILOS/issues/2265
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)